### ISSUE TITLE
Stretch Rest for Elves

### ISSUE THREAD

**Issue by abeven-sudo** [pafvel/dragonbane#68]
Elves only roll 1d6 to heal on a stretch rest. Inner Peace should allow them to roll 2D6 instead.

**Comment by pafvel** [comment]
Abilities are generally not automated. Hence classifying this as an enhancement rather than a bug.

**Comment by abeven-sudo** [comment]
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/3a8bde72-f858-4e27-9bd2-39ecc3032970">

I implemented as such...

**Comment by pafvel** [comment]
Strictly speaking it's not correct because it is the Kin Ability that matters, but in practice I guess it works.

**Comment by abeven-sudo** [comment]
Well to be fair I am bad at coding AND logic.

**Comment by bb46003** [comment]
maby it can be done by AE ? so if character have kin Elf it add passive AE wich add extra d6 to recovering HP and WP

**Comment by Fabbe** [comment]
I think it has to be player activated or at least popup in the chat for the player when the player clicks for short rest.
I would prefer the latter I think.

**Comment by Kergalli** [comment]
This is an old one, and I have been mainly handling Elf Stretch Rests via macro when I have an Elf PC, but the suggestion from @abeven-sudo is solid. I just adjust my actor.js to be:

```
  async restStretch() {
    await this.update({ ["system.canRestStretch"]: false });

    let numdice = "1";

    // Adjust for Inner Peace kin ability
    if (
      this.items.some(
        (item) => item.name === "Inner Peace" && item.type === "ability"
      )
    ) {
      numdice = "2";
    }
```

**Comment by Kergalli** [comment]
Absolutely. Needs a more robust/localized approach for production, but as a quick fix the hardcoded language can be adjusted for whatever language you are using.
